src/screens/LocationPage/LocationPage.tsx

```diff
@@ -33,7 +33,7 @@ function WithSearchInNav({ region }: LocationPageProps) {
         hasScrolled={hasScrolled}
         region={region}
       />
-      <ChartsHolder chartId={chartId} region={region} />
+      <ChartsHolder chartId={chartId} region={region} isLocationPage={true} />
```

Comment:
we dont need this `isLocationPage` prop. the presence of a region lets you know that its a location page